initfiles/componentfiles/configxml/sparkThor.xsd

```diff
@@ -32,7 +32,7 @@
                 </xs:appinfo>
             </xs:annotation>  
             <xs:sequence>
-                <xs:element name="Instance" maxOccurs="unbounded">
+                <xs:element name="SparkMasterInstance" maxOccurs="unbounded">
```

Comment:
Limit this to a single instance. Also, are spaces allowed? If so, "Spark Master Instance" ?

Reply:
Spaces aren't allowed.  Tried that but it's not valid unfortunately.